webui/react/src/pages/TrialDetails.tsx

```diff
@@ -302,9 +339,29 @@ If the problem persists please contact support.',
 
   useEffect(() => {
     if (experimentId === undefined) return;
-    getExperimentDetails({ id:experimentId })
-      .then(experiment => setExperiment(experiment));
-  }, [ experimentId ]);
+
+    const fetchExperimentDetails = async () => {
+      try {
+        const response = await getExperimentDetails({ id: experimentId });
+        setExperiment(response);
+
+        const storageMetricsKey = `experiments/${response.id}/STORAGE_METRICS_KEY`;
```

Comment:
did you mean to use the constant STORAGE_METRICS_KEY here?

Reply:
doh!